audit/projects/k8s-staging-ci-images/iam.json

```diff
@@ -0,0 +1,68 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "serviceAccount:731599680865@cloudbuild.gserviceaccount.com",
+        "serviceAccount:deployer@k8s-prow.iam.gserviceaccount.com",
+        "serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
+      ],
+      "role": "roles/cloudbuild.builds.builder"
+    },
+    {
+      "members": [
+        "group:k8s-infra-staging-ci-images@kubernetes.io"
+      ],
+      "role": "roles/cloudbuild.builds.editor"
+    },
+    {
+      "members": [
+        "serviceAccount:service-731599680865@gcp-sa-cloudbuild.iam.gserviceaccount.com"
+      ],
+      "role": "roles/cloudbuild.serviceAgent"
+    },
+    {
+      "members": [
+        "serviceAccount:service-731599680865@container-analysis.iam.gserviceaccount.com"
+      ],
+      "role": "roles/containeranalysis.ServiceAgent"
+    },
+    {
+      "members": [
+        "serviceAccount:k8s-infra-gcr-vuln-scanning@k8s-artifacts-prod.iam.gserviceaccount.com"
+      ],
+      "role": "roles/containeranalysis.occurrences.viewer"
+    },
+    {
+      "members": [
+        "serviceAccount:service-731599680865@containerregistry.iam.gserviceaccount.com"
+      ],
+      "role": "roles/containerregistry.ServiceAgent"
+    },
+    {
+      "members": [
+        "serviceAccount:service-731599680865@gcp-sa-containerscanning.iam.gserviceaccount.com"
+      ],
+      "role": "roles/containerscanning.ServiceAgent"
+    },
+    {
+      "members": [
+        "user:davanum@gmail.com"
```

Comment:
All LGTM I think, it took some time to go over all the files related, but I did not see anything that calls my attention

just one comment, in several iam.json files we only see one `"roles/owner"` it is not better to have at least one more for backup in case it is needed? or this is expected? (asking because might be discussed before to be like this)

Reply:
Whoever runs the script that creates a project gets assigned as owner (we have a bug open about this https://github.com/kubernetes/k8s.io/issues/299).  Org owners implicitly get owner on projects within the org, so we're available for fallback.

I believe the thinking goes: don't give owner access to projects, thereby requiring a gitops workflow for things like changing iam permissions.  It's just unfortunate that the execution part of our gitops model is currently "humans run scripts" vs. bots.

Reply:
thanks for the clarification Aaron